src/langapi/language_ui.cpp

```diff
@@ -31,24 +31,14 @@ bool language_uit::parse()
 
 bool language_uit::parse(const std::string &filename)
 {
-  int mode = get_mode_filename(filename);
+  int mode = get_mode_filename(filename, &(config.options));
```

Comment:
I think that config is a global variable already

Reply:
yeah, no need to pass it around.